src/openforms/config/templatetags/privacy_policy.py

```diff
@@ -10,11 +10,9 @@
 def privacy_policy():
     conf = GlobalConfiguration.get_solo()
     if conf.privacy_policy_url:
-        template_string = """
-        {% load i18n %}<a href="{{ privacy_policy }}" target="_blank" rel="noreferrer noopener">
-            {% trans "privacy policy" %}
-        </a>
-        """
+        template_string = """{% load i18n %}
+        <a href="{{ privacy_policy }}" target="_blank" rel="noreferrer noopener">{% trans "privacy policy" %}</a>"""
+
```

Comment:
I find it a little bit finicky and hurting readability - this can easily be refactored again by someone else. It's also a bit overkill maybe to use the template engine here (again), so perhaps refactor to `format_html`?

```py
from django.utils.html import format_html

tpl = '<a href="{}" target="_blank" rel="noreferrer noopener">{}</a>'
return format_html(
    tpl,
    conf.privacy_policy_url,
    _("privacy policy"),
)
```